src/js/player.js

```diff
@@ -1168,13 +1168,9 @@ Player.prototype.selectSource = function(sources){
  * @return {String} The current video source when getting
  * @return {String} The player when setting
  */
-Player.prototype.src = function(source){
+Player.prototype.src = function(source=this.techGet('src')){
```

Comment:
There's elements of both that I like so I don't have a strong opinion here, but for other default args in this PR it has no spaces. We should be consistent either way.